rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/synonyms/110_synonyms_invalid.yml

```diff
@@ -14,10 +14,8 @@ setup:
   # This is to ensure that all index shards (write and read) are available. In serverless this can take some time.
   - do:
       cluster.health:
-        index: .synonyms-2
-        timeout: 2s
+        index: .synonyms
```

Comment:
Not related to this PR, but for my educaiton: what is `.synynoms-2`? Why we added it?

Reply:
`.synonyms-2 `is the current version of the index - we changed the index format before releasing it, so it has a `-2` suffix to it.

`.synonyms` is the alias for the system indices. I think it's more correct to refer to the alias instead of the specific index, so in case it changes we don't have to change all the YAML tests.

So, it's unrelated to this change, but I thought we could do some winter cleaning here 🙂